.gitlab-ci.yml

```diff
@@ -69,7 +69,7 @@ variables:
   RELEASE_VERSION_7: nightly
   DATADOG_AGENT_BUILDIMAGES: v2028839-bd64f94
   DATADOG_AGENT_BUILDERS:  v2028983-aee8904
-  DATADOG_AGENT_WINBUILDIMAGES: v2021523-34e07eb
+  DATADOG_AGENT_WINBUILDIMAGES: v2068298-dc0e899
```

Comment:
once you merge `datadog-agent-builders`, a new tag will be generated from master